rfcs/rfc1/README.md

```diff
@@ -0,0 +1,211 @@
+
+4. Submit a pull request.
+
+5. Each pull request will automatically be labeled as `Draft` to allow for collaborative refinement of the proposal.
+
+6. The author should be prepared to integrate feedback and build consensus. RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
+
+7. RFCs rarely go through this process unchanged, especially as alternatives and drawbacks are shown.  You can make edits, big and small, to the RFC to clarify or change the design, but make changes as new commits to the pull request, and leave a comment on the pull request explaining your changes.  Specifically, do not squash or rebase commits after they are visible on the pull request.
+
+8. When the RFC has been sufficiently discussed and refined, it is considered ready for review by the Technical Steering Committee (TSC).  The author should label the RFC as `Proposed` so that it will be scheduled for discussion at the next TSC weekly meeting.
+
+### <ins>Decision making</ins>
+
+Once the RFC is labeled as `Proposed`, the following will happen at the next TSC weekly meeting:
+
+1. An author of the RFC is expected to be present and becomes the RFC DRI
+
+2. The `Proposed at` timestamp is set to the meeting date
+
+3. 10 minutes of silent reading
+
+4. 10 minutes of Q&A with a focus on understanding what is being proposed, not to provide feedback
+
+5. After the meeting, TSC members review the RFC in further detail, discuss with impacted stakeholders, and post feedback to the RFC pull request
+
+6. TSC members are expected to respond within 7 days (or less) to either:
+
+   * Provide blocking feedback
+   
+   * Accept the proposal (with or without feedback)
+
+7. The RFC DRI is expected to address feedback by either changing the RFC or by explaining.
+
+8. At the next TSC weekly meeting, members review the feedback and:
+
+   * For Type 1 decision making: If there is no blocking feedback, the RFC is accepted
+
+   * For Type 2 decision making: If all TSC members accept, the RFC is accepted
+
+   * If the TSC reaches an impasse, the TSC lead will make the decision
+
+### <ins>After the decision</ins>
+
+If the RFC is not accepted, the author(s) can propose it again after addressing blocking feedback.
+
+Accepted RFCs will be marked as "Active" and a TSC member will be assigned as the Directly Responsible Individual (DRI) for overseeing the implementation of the RFC.  The DRI can delegate the implementation responsibilities to another individual or team. The RFC DRI will work with TBD Engineering Leads to prioritize and staff the implementation.
+
+Every accepted RFC has an associated issue tracking its implementation in the _____ repository; thus that associated issue can be assigned a priority via the triage process that the TBD Engineering team uses.  Community members who are interested in working on the implementation for an "active" RFC can volunteer by leaving a comment on the associated issue.
```

Comment:
`respective repository` meant to go here?

Reply:
https://github.com/TBD54566975/technical-steering-committee/pull/1/files#r1612348345